source/nodejs/adaptivecards-designer/src/designer-peers.ts

```diff
@@ -302,14 +304,21 @@ export class StringPropertyEditor extends SingleInputPropertyEditor {
             return [
                 {
                     caption: "...",
+                    altText: (this.label + " " + "Data Binding Collapsed"),
```

Comment:
`collapsed`/`expanded` should be represented using the [`aria-expanded`](https://w3c.github.io/aria/#aria-expanded) attribute.